spec.html

```diff
@@ -41791,9 +41791,10 @@ <h1>%TypedArray%.prototype.copyWithin ( _target_, _start_ [ , _end_ ] )</h1>
             1. Set _taRecord_ to MakeTypedArrayWithBufferWitnessRecord(_O_, ~seq-cst~).
             1. If IsTypedArrayOutOfBounds(_taRecord_) is *true*, throw a *TypeError* exception.
             1. Set _len_ to TypedArrayLength(_taRecord_).
+            1. NOTE: Side-effects of the above steps may have reduced the size of _O_, in which case copying should proceed with the longest still-applicable prefix.
+            1. Set _count_ to min(_count_, _len_ - _targetIndex_, _len_ - _startIndex_).
```

Comment:
```suggestion
            1. Set _count_ to min(_count_, _len_ - _startIndex_, _len_ - _targetIndex_).
```

Very minor nit. The `min` call before we enter the `count > 0` conditional does the _startIndex_ case first and _targetIndex_ case second.

Reply:
Done.